README.md

```diff
@@ -2,6 +2,8 @@
 
 Scripts used in our official [Raspberry Pi power button guide](https://howchoo.com/g/mwnlytk3zmm/how-to-add-a-power-button-to-your-raspberry-pi).
 
+I added events handling for shutdown + reboot and a status LED.
```

Comment:
Let's remove this. If this gets merged, I can update the original power button guide to include instructions for adding the status LED.